daml-assistant/integration-tests/src/Main.hs

```diff
@@ -160,7 +172,12 @@ quickstartTests quickstartDir mvnDir = testGroup "quickstart"
                   (socket (addrFamily addr) (addrSocketType addr) (addrProtocol addr))
                   close
                   (\s -> connect s (addrAddress addr))
-    , testCase "mvn compile" $
+    ] <>
+    -- The mvn tests seem to fail on Windows for some reason so for now we disable them.
```

Comment:
Can we get a ticket to have this fixed at some point?

Reply:
Opened a ticket and linked to it from this comment.